config.md

```diff
@@ -265,6 +265,27 @@ Annotations are key-value maps.
 }
 ```
 
+## HypervisorPath
+
+HypervisorPath is an optional absolute path to the hypervisor binary used by VM-based runtimes.
```

Comment:
Possibly, but I think it depends on whether the runtime should be making decisions, or whether it should just do what it's told by acting on the contents of config.json.

Since that runtime is running in a potentially restricted environment and having its actions directed by entities such as docker/containerd, I've opted for the latter imperative approach. I think that is also potentially safer since those higher-level entities can potentially perform the appropriate validity checks before handing the config to the runtime (see docker/docker#21324).

Ultimately, I'd prefer that the runtime avoid having to use heuristics, PATH searching and custom-config files to allow it to handle the workload. By encoding all configuration the runtime needs in the OCI configuration, it should be fully portable to any OCI-compliant environment.

If we imagine that the VM configuration does not form part of the OCI spec, the runtime would then have to find its custom config file (to know which hypervisor to use, etc).  This might not be as trivial as it sounds given that various distros treat config file locations differently. That's not even considering the Windows aspect.